cico_setup.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -ex
+
+load_jenkins_vars() {
+    if [ -e "jenkins-env" ]; then
+        cat jenkins-env \
+          | grep -E "(DEVSHIFT_TAG_LEN|DEVSHIFT_USERNAME|DEVSHIFT_PASSWORD|JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId)=" \
+          | sed 's/^/export /g' \
+          > ~/.jenkins-env
+        source ~/.jenkins-env
+    fi
+}
+
+prep() {
+    yum -y update
+    yum -y install docker rpmdevtools copr-cli
+    systemctl start docker
+}
+
+run_tests() {
+    make check
+}
+
+build_rpm() {
+    rpmdev-setuptree
+    rm -f ~/rpmbuild/SRPMS/mercator-*.src.rpm
+    ./make_rpm.sh --source
+    copr-cli build jpopelka/mercator ~/rpmbuild/SRPMS/mercator-*.src.rpm
```

Comment:
as I wrote in almighty repository, you should create a file where credentials file for copr-cli here.